stellargraph/core/graph_networkx.py

```diff
@@ -772,6 +772,30 @@ def to_adjacency_matrix(self, nodes: Optional[Iterable] = None):
             self._graph, dtype="float32", weight=self._edge_weight_label, format="coo"
         )
 
+    def to_networkx(self):
+        # Despite this class using NetworkX, this implementation does not directly use that
+        # representation, so that it can be reused as we move away from being NetworkX-based.
+        if self.is_directed():
+            graph = nx.MultiDiGraph()
+        else:
+            graph = nx.MultiGraph()
+
+        types = self.node_types
+
+        for ty in types:
+            node_ids = self.nodes_of_type(ty)
+            features = self.node_features(node_ids, node_type=ty)
```

Comment:
Results  in error when the nodes have no features.

Reply:
Good catch 👍